examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -202,41 +253,85 @@ def apply(inputs):
         bounding_boxes = keras_cv.bounding_box.convert_format(
             bounding_boxes, images=image, source="yxyx", target=bounding_box_format
         )
-        return {"images": image, "bounding_boxes": bounding_boxes}
+        return image, bounding_boxes
 
     return apply
 
 
+def proc_eval_fn(bounding_box_format, target_size):
+    def apply(inputs):
+        raw_image = inputs["image"]
+        raw_image = tf.cast(raw_image, tf.float32)
+
+        img_size = tf.shape(raw_image)
+        height = img_size[0]
+        width = img_size[1]
+
+        target_height = tf.cond(
+            height > width,
+            lambda: 640.0,
```

Comment:
Probably lift this 640 out as a constant.

Reply:
Done.